### PR TITLE
Jwshuff/dependencies correction

### PR DESCRIPTION
**Pull request by JWShuff**

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'spec_helper'
+require "view_component/test_helpers"
+require "capybara/rspec"
```
Comment (patrickarnett) [inline spec/rails_helper.rb:4]: Are these necessary? Tests pass for me without it, but maybe I'm missing something?

**Comment by patrickarnett** [comment]
@JWShuff Sorry, I meant to open a PR against this branch but ended up pushing to it instead. How does [this commit](https://github.com/patrickarnett/view_component-fragment_caching/pull/2/commits/d6dc86b71383941bac4295931c238b20a3a79d92) look?

File: spec/rails_helper.rb (hunk 1 above)
Comment (JWShuff) [reply spec/rails_helper.rb:4]: They are unnecessary, was when we were trying to sort out what was going wrong with the templating lookup before the current implementation

**Comment by JWShuff** [comment]
Looks good, I think it cleans up the digestor nicely.

**Pull request merged**